lwallet/Command.hs

```diff
@@ -27,10 +28,12 @@ import           Pos.Types                  (Address (..), BlockVersion, Coin, E
                                              SoftwareVersion, decodeTextAddress, mkCoin)
 import           Pos.Update                 (SystemTag, UpId, mkSystemTag)
 
+data SendMode = SendNeighbours | SendRoundRobin | SendRandom deriving Show
```

Comment:
Can you write a comment about this type? It's not obvious for unprepared reader.